src/DocumentFragment.js

```diff
@@ -0,0 +1,38 @@
+  // ### DocumentFragment Shim
+  //
+  // A representation of a document fragment that can be normally generated
+  // by calling `document.createDocumentFragment`. Since the document fragment
+  // is itself a node, it inherits all properties of a node.
+module.exports = (function(){
+  "use strict";
+
+  var Node = require('./Node');
+
+  function DocumentFragment() {
+    Node.call(this, Node.DOCUMENT_FRAGMENT_NODE, "#document-fragment");
```

Comment:
do we actually need this? Does the code actually *depend* on having objects with Node's prototype?